api/webhook/github_push_test.ts

```diff
@@ -77,6 +80,40 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "push event forbidden name",
+  async fn() {
+    // Send push event
+    const resp = await handler(
+      createJSONWebhookEvent(
+        "push",
+        "/webhook/gh/frisbee",
+        pusheventforbidden,
+        { name: "frisbee" },
+        {},
+      ),
+      createContext(),
+    );
+    assertEquals(resp, {
+      body:
+        '{"success":false,"error":"found inapropriate word in module name"}',
+      headers: {
+        "content-type": "application/json",
+      },
+      statusCode: 400,
+    });
+
+    // Check that no versions.json file exists
+    assertEquals(await getMeta("testing", "versions.json"), undefined);
```

Comment:
`testing` should be `firsbee` right?

Reply:
yup (that's what I get for keeping the tests for last 🤦 )